calc_exercise.rb

```diff
@@ -0,0 +1,56 @@
+# August 7th
+
+# definition of operators
+def calculator_operators
+  puts "1. add (+)"
+  puts "2. subtract (-)"
+  puts "3. mulitply (*)"
+  puts "4. divide (/)"
+end
+
+# Verifying valid operators have been selected
+def valid_num(number)
+  until (true if Float(number) rescue false)
```

Comment:
In general, this reads kind of funny. Also, `number` will always be a float, since you get `number` from the call `gets.chomp.to_f`, which converts them to floats all the time.

Overall, the code works, but is just a little funny -- I wouldn't call this a high priority thing to correct, but something to return to in a few days or weeks with fresh eyes.